include/capture_util.h

```diff
@@ -0,0 +1,18 @@
+#ifndef GUARD_CAPTURE_UTIL_H
+#define GUARD_CAPTURE_UTIL_H
+
+struct BallData
+{
+    u16     multiplier;
+    u16     divider;
+    bool8   guaranteedCapture;
+    u8      flatBonus;
```

Comment:
```suggestion
    u16 multiplier;
    u16 divider;
    bool8 guaranteedCapture;
    u8 flatBonus;
```
I don't think we normally align members.